.babelrc.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @HACK: Babel doesn't support using modules as your `babelrc`.
```

Comment:
Maybe just edit `.babelrc` instead of new file? [Like my usage](https://github.com/jhen0409/react-native-debugger/blob/master/webpack/base.babel.js).

Also, we can waiting babel 7 for [`.babelrc.js` support](https://github.com/babel/babel/pull/4892).